Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,628 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
 | Heerlen           |
 | Alkmaar           |
 +-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+28 rows in set (0.00 sec)
+
+ select Population,Name from city where Name='Rotterdam';
++------------+-----------+
+| Population | Name      |
++------------+-----------+
+|     593321 | Rotterdam |
++------------+-----------+
+1 row in set (0,00 sec)
+
+3
+
+mysql> select Name ,continent from country where continent = 'North America';
```

Comment:
Be very careful with your capital letters here. In databases, same as variables when programming, `Continent` may not be the same as `continent`.